crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/service/stateless/StatelessAssessmentService.java

```diff
@@ -0,0 +1,148 @@
+        return new StatelessResult(null, initialAssessment);
+    }
+
+    private StatelessInitialResult initialResult(@NotNull LocalDateTime date,
+                              boolean hasPartner,
+                              CaseType caseType,
+                              MagCourtOutcome magCourtOutcome,
+                              @NotNull BigDecimal totalIncome,
+                              @NotNull Map<AgeRange, Integer> childGroupings) {
+        final var criteriaEntry = assessmentCriteriaService.getAssessmentCriteria(date, hasPartner, false);
+
+        final var children = convertChildGroupings(childGroupings, criteriaEntry.getAssessmentCriteriaChildWeightings());
+
+        // assessmentStatus has to be set 'COMPLETE' otherwise the return value is null
+        final MeansAssessmentRequestDTO requestDTO = MeansAssessmentRequestDTO
+                .builder()
+                .childWeightings(children)
+                .assessmentStatus(CurrentStatus.COMPLETE)
+                .build();
+
+        final var service = new InitMeansAssessmentService(childWeightingService);
+        final var result = service.execute(totalIncome, requestDTO, criteriaEntry);
+        final var fullAssessmentPossible = new FullAssessmentAvailabilityService()
+                .isFullAssessmentAvailable(caseType, magCourtOutcome, null, result.getInitAssessmentResult());
+
+        return new StatelessInitialResult(
+                result.getInitAssessmentResult(),
+                criteriaEntry.getInitialLowerThreshold(),
+                criteriaEntry.getInitialUpperThreshold(),
+                fullAssessmentPossible
+        );
+    }
+
+    public static BigDecimal incomeTotals(AssessmentCriteriaService assessmentCriteriaService,
+                                          AssessmentCriteriaEntity assessmentCriteria,
+                                          @NotNull List<Income> incomes) {
+        var sectionSummaries = DataAdapter.incomeSectionSummaries(assessmentCriteria, incomes);
+        var requestDTO = MeansAssessmentRequestDTO
+                .builder()
+                .sectionSummaries(sectionSummaries)
+                .build();
+        return calculateSummariesTotal(assessmentCriteriaService, requestDTO, assessmentCriteria);
+    }
+
+    public static BigDecimal outgoingTotals(AssessmentCriteriaService assessmentCriteriaService,
+                                            AssessmentCriteriaEntity assessmentCriteria,
+                                            @NotNull List<Outgoing> outgoings) {
+        var sectionSummaries = DataAdapter.outgoingSectionSummaries(assessmentCriteria, outgoings);
+        var requestDTO = MeansAssessmentRequestDTO
+                .builder()
+                .sectionSummaries(sectionSummaries)
+                .build();
+        return calculateSummariesTotal(assessmentCriteriaService, requestDTO, assessmentCriteria);
```

Comment:
I couldn't work out how to do it w/o losing the types. I felt that this was a good compromise rather than making everything a string, which was my first attempt